docs/source/cpp/compute.rst

```diff
@@ -1343,6 +1343,8 @@ null input value is converted into a null output value.
 
 .. _detailed formatting documentation: https://howardhinnant.github.io/date/date.html#to_stream_formatting
 
+* \(2) The strptime kernel currently ignores the ``%Z`` specifier for any string.
```

Comment:
I wonder if the "for any string" is fully clear (I am also not a good wordsmith, so not directly an alternative)

Something more verbose: ".. ignores the %Z specifier and any string at the matching location" ?

Reply:
"The strptime kernel presently overlooks the "%Z" specifier along with any string located at the corresponding position within the input." maybe this?  @jorisvandenbossche

Reply:
Sorry this stalled @Divyansh200102! We sometimes loose track of waiting PRs. I think we should proceed with this, let's just agree on the wording.

> "The strptime kernel presently overlooks the "%Z" specifier along with any string located at the corresponding position within the input."

Seems ok to me. What do you think @jorisvandenbossche ?